### PR TITLE
pelux.xml: bump to rocko

### PR DESCRIPTION
**Pull request by tobsan**
Signed-off-by: Tobias Olausson <tobias.olausson@pelagicore.com>

### CODE REVIEW

File: pelux.xml [hunk 1]
```diff
@@ -12,66 +12,70 @@
...
-           revision="717303e6fbcbbe181ad9645d762eb5a85d934523"
+           upstream="rocko"
+           revision="a03c452917842b15451ad7098c1216eaec09cd2b"
            name="poky"
            path="sources/poky"/>
 
   <project remote="github"
-           revision="28f52b1451c6bebe46224b81dc6849babf286ce1"
-           upstream="pyro"
+           revision="rocko"
```
Comment (thenor) [inline pelux.xml]: Shouldn't this point to a commit ID?
Same for meta-pelux? Or do you want them to not be connected to a commit for now?

Comment (tobsan) [reply pelux.xml]: This is stated in the commit message :)

**Comment by tobsan** [comment]
Let's wait for the CI to finish. It already caught one major issue so I would like to be sure.

**Comment by tobsan** [comment]
After discussions on IRC, updated so that we don't follow latest in our own layers, since we don't have the CI set up to handle that properly.

**Pull request merged**